test/integration/other/transactions.js

```diff
@@ -46,7 +47,7 @@ describe('enketo worker transaction', () => {
     const { Audits, Forms, oneFirst } = container;
 
     try {
-      const simple = (await Forms.getByProjectAndXmlFormId(1, 'simple')).get();
+      const simple = (await Forms.getByProjectAndXmlFormId(1, 'simple', Form.AnyVersion)).get();
```

Comment:
Does this need to be Form.AnyVersion? I think it can be Form.WithoutDef. The audit logging only really cares about the acteeId of the form and the Forms.update call here is similarly only working with the top level form record, not the def.

Reply:
Done :+1: